polkadot/node/network/availability-recovery/src/lib.rs

```diff
@@ -1098,19 +423,73 @@ async fn handle_recover<Context>(
 
 	let _span = span.child("session-info-ctx-received");
 	match session_info {
-		Some(session_info) =>
+		Some(session_info) => {
+			let mut prefer_backing_group = true;
```

Comment:
> I think it would be more readable to call it let mut pov_is_small: Option<bool> = None; where None means we don't know.

I tried rewriting it as you suggested, but I think it's less readable IMO. I think it's more descriptive as `prefer_backing_group `. From just looking at the variable name, you can tell how it's going to be used and its purpose. But maybe I misunderstood your suggestion?

> I am not sure we should default to true in that case like we do now.

Indeed, I thought about this too. I preserved the previous behaviour in this PR, which is why I defaulted to true.
I think we should default to false if the strategy is `BackersFirstIfSizeLower` and we're unable to get the chunk size, but default to true for strategies that don't take into account the pov size, like we currently do.